cfme/tests/openstack/cloud/test_volume_backup.py

```diff
@@ -64,6 +67,60 @@ def incremental_backup(volume_backup, provider):
         logger.warning('Exception during volume backup deletion - skipping..')
 
 
+@pytest.fixture(scope='function')
+def new_instance(provider):
+    prov_data = provider.data['provisioning']
+    prov_form_data = {
+        'request': {'email': fauxfactory.gen_email(),
+                    'first_name': fauxfactory.gen_alpha(),
+                    'last_name': fauxfactory.gen_alpha()},
+        'catalog': {'num_vms': '1',
+                    'vm_name': fauxfactory.gen_alpha()},
+        'environment': {'cloud_network': prov_data['cloud_network'],
+                        'cloud_tenant': prov_data['cloud_tenant']},
+        'properties': {'instance_type': partial_match(prov_data['instance_type'])},
+    }
+
+    instance_name = prov_form_data['catalog']['vm_name']
+
+    try:
+        instance = provider.appliance.collections.cloud_instances.create(
+            instance_name,
+            provider,
+            prov_form_data, find_in_cfme=True
+        )
+
+    except KeyError:
+        # some yaml value wasn't found
+        pytest.skip('Unable to find an image map in provider "{}" provisioning data: {}'
+                    .format(provider, prov_data))
+
+    yield instance
+
+    instance.cleanup_on_provider()
+
+
+@pytest.fixture(scope='function')
+def incremental_backup_of_attached_volume(appliance, provider, new_instance, volume_backup):
+    volume = volume_backup.appliance.collections.volumes.instantiate(volume_backup.volume, provider)
```

Comment:
Required:
If you have `appliance` object then no need to use the `appliance`  attribute  of `volume_backup`.